ui/lib/kv/addon/components/page/secret/paths.hbs

```diff
@@ -25,7 +25,7 @@
       >
         <Icon @name="clipboard-copy" aria-label="Copy" />
       </CopyButton>
-      <code class="has-left-margin-s level-left">
+      <code class="is-flex-1 text-overflow-ellipsis has-left-margin-s" truncate-value>
```

Comment:
And up on the `InfoTableRow` on row 18 you would pass in `truncateValue={{true}}`